readme.txt

```diff
@@ -56,4 +56,28 @@ To report a security issue, please visit the [WordPress HackerOne](https://hacke
 
 == Changelog ==
 
-See the [release history](https://github.com/wordpress/two-factor/releases).
+= 0.13.0 - 2025-04-02 =
+- Add two_factor_providers_for_user filter to limit two-factor providers available to each user by @kasparsd in #669
+- Update automated testing to cover PHP 8.4 and default to PHP 8.3 by @BrookeDot in #665
+
+= 0.12.0 - 2025-02-14 =
+- Simplify the Two Factor settings in user profile by @kasparsd in #654

```

Comment:
On GitHub the author and pull request number would get linked automatically. I wonder if we should do that here too. Having that data without the links really doesn't help users to quickly see what is included.